runsc/container/state_file.go

```diff
@@ -385,6 +385,9 @@ func (s *StateFile) load(v any, opts LoadOpts) error {
 
 	metaBytes, err := os.ReadFile(s.statePath())
```

Comment:
path := s.statePath()
metaBytes, err := os.ReadFile(path)
if err != nil {
	// Caller of this function relies on error code, we cannot return new error,
	// but we need to provide a message with file name.
	log.Warningf("Error loading state file %q: %v", path, err)
	return err
}